src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_ActiveFareProductsTabScreen.tsx

```diff
@@ -32,7 +34,13 @@ export const TicketTabNav_ActiveFareProductsTabScreen = () => {
         reservations={reservations}
         fareContracts={activeFareContracts}
         isRefreshing={isRefreshingFareContracts}
-        refresh={resubscribeFirestoreListeners}
+        refresh={() => {
+          resubscribeFirestoreListeners();
+          analytics.logEvent('Ticketing', 'Pull to refresh tickets', {
+            reservationsCount: reservations.length,
+            activeFareContractsCount: activeFareContracts.length,
+          });
+        }}
```

Comment:
I think this should use `useCallback`. Kinda risky to recreate the function on every render, as we don't know how it can be used inside the child component.